--- origsrc/SDL_Perl-2.1.3/make/lib/SDL/Build/Cygwin.pm	2005-10-05 04:25:49.000000000 -0500
+++ src/SDL_Perl-2.1.3/make/lib/SDL/Build/Cygwin.pm	2009-04-29 02:19:53.389000000 -0500
@@ -28,17 +28,4 @@ sub fetch_includes
 	);
 }
 
-sub build_links
-{
-	my $self  = shift;
-	my $links = $self->SUPER::build_links();
-
-	for my $subsystem (values %$links)
-	{
-		push @{ $subsystem{ libs } }, '-lpthreads';
-	}
-
-	return $links;
-}
-
 1;
